src/components/Navbar/MobileNavbar.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Box, IconButton, makeStyles, Menu, Typography } from '@material-ui/core';
+import clsx from 'clsx';
+import { CFC } from '../../typings/components';
+import { NavbarProps } from './Navbar';
+import NavItems from './NavItems';
+import { Menu as MenuIcon } from 'react-feather';
+
+type MobileNavbarProps = Omit<NavbarProps, 'type' | 'resumeLink'>;
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    padding: theme.spacing(1, 2),
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: '0 6px 16px 0 rgba(0, 0, 0, 0.1)',
+    overflow: 'hidden',
+  },
+  image: {
+    width: 40,
+    height: 40,
+    borderRadius: 8,
+    marginRight: theme.spacing(1),
+  },
+  menu: {
+    padding: 0,
+
+    '& .MuiMenuItem-root': {
+      minWidth: 160,
+      justifyContent: 'flex-start',
+    },
+  },
+}));
+
+export const MobileNavbar: CFC<MobileNavbarProps> = ({ className, fullName, image, position, ...props }) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const classes = useStyles();
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <Box {...props} className={clsx(className, classes.root)}>
+        <Box display="flex" alignItems="center">
```

Comment:
Out of curiosity, is there a benefit for using style props from Box instead of classes? Or when to use which? Thanks 😄

Reply:
There is one - less code. And if you have your own components library, you generally don't need to create custom styles in views. You are only using spacing and layout related styles to put already styled components into the right places. In the company, we have a rule that for every component that needs less than four custom styles, we are using props for styling.